docs/reference/function/add_logger.txt

```diff
@@ -26,7 +26,7 @@ Definition
 Parameters
 ----------
 
-``$logger`` : Psr\\Log\\LoggerInterface
+``$logger`` : Psr\Log\LoggerInterface
```

Comment:
@norareidy: Correct me if I'm wrong, but `\\` should only exist in title headings. I assume the plain text here should be changed to remove the escaping.

Reply:
After staging the docs for this PR, it looks like the `\\` should stay in this case as the single backslash doesn't show up -[ here's the staging link](https://preview-mongodbnorareidy.gatsbyjs.io/php-library/DOCSP-36627-v1.17/reference/function/add_logger/) to this page.